modules/ROOT/pages/features/UserDefinedFunctions.adoc

```diff
@@ -98,9 +98,33 @@ As an input it receives a SIP3 packet with generic payload represented as a `Map
 
 As mentioned in the previous section, the main purpose of any SIP3 UDF is assigning `String` or `Boolean` attribute to a SIP3 packet according to its business logic.
 
-We call such attributes User-Defined Attributes (UDAs). UDAs can bring your SIP3 experience to a next level if you follow the following rule:
+We call such attributes User-Defined Attributes (UDAs). Each UDA by default will be provisioned as an xref::features/AdvancedSearch.adoc#_expressions[Advanced Search expression] and as a xref::features/ServiceMetrics.adoc[Metrics tag].
 
-IMPORTANT: Always choose `String` attributes with a limited amount of possible values. Otherwise, it may affect your database performance and can even bring it down. For instance, if you have a mobile application: `operation_system` or `application_version` are good choice of UDAs.
+However, in some cases you may want to have more control over your UDAs provisioning by using a special SIP3 UDAs syntax in attributes names.
+
+- _Database_ (*d*) - in this mode your attribute will be saved to a database and provisioned as an Advanced Search expression.
+- _Options_ (*o*) - enables autocompletion for Advanced Search expression.
+- _Metrics_ (*m*) - in this mode your attribute will be saved to a database and provisioned as a Metrics tag.
```

Comment:
I believe this description is incorrect and attribute will be provisioned as a Metrics tag only